### ISSUE TITLE
Style Settings placed in plugins styles.css not loading

### ISSUE THREAD

**Issue by nothingislost** [mgmeyers/obsidian-style-settings#22]
I'm creating Style Settings within my plugin's styles.css and they do not appear to render within the Style Settings pane on app load. The only way to get them to show up is to make a change that triggers CSS change.

After figuring that out, I added this to my onload and that fixed the problem `this.app.workspace.trigger("css-change");`

This may be an order of operations issue as I do see that you also call `this.app.workspace.trigger("css-change");` within your plugin.

Not many plugins are using Style Settings yet but I do see the same behavior with the Buttons plugin.

**Comment by hieu6** [comment]
THANK YOU for posting this!

**Comment by mgmeyers** [comment]
@hieu6 Yes, adding a CSS file with a style settings config to your snippets folder will work

**Comment by mgmeyers** [comment]
@nothingislost I've made a small modification that should hopefully help with this. Can you test out the latest version and see if your plugin style settings load as desired?
